mrp_production_real_cost/models/mrp_production_workcenter_line.py

```diff
@@ -97,7 +97,7 @@ def action_pause(self):
 
     @api.multi
     def action_done(self):
+        result = super(MrpProductionWorkcenterLine, self).action_done()
```

Comment:
This order change is because machine uptime is calculated with `start_date` and `end_date` if we create the analytic line before uptime will be 0, as `end_date` won't be defined (if the workorder has not been previously paused)